docs/source/_templates/_static/css/ignite_theme.css

```diff
@@ -143,3 +143,7 @@ article.pytorch-article table.longtable.docutils.align-default tbody td p {
   margin-block-start: 0.1em;
   margin-block-end: 0.1em;
 }
+
+article.pytorch-article div#ignite-contrib-handlers.section div#time-profilers.section dl.py.class dd dl.py.method dd div.highlight-text.notranslate div.highlight pre {
+  font-size: 62%;
```

Comment:
Maybe, 75% will be enough and not that small :)

Reply:
the problem is 62% is the highest acceptable value to fit line, 75 will break the lines

Reply:
75%
![Screenshot from 2020-10-31 19-37-12](https://user-images.githubusercontent.com/30748114/97785907-8b49dd80-1bb0-11eb-978c-03856a486521.png)

Reply:
There is another idea, to trim that example a little, but keep all important information:

```diff

            -------------------------------------  -------------------  -------------- ...
            Handler                                Event Name                 Total(s)
            -------------------------------------  -------------------  --------------
            run.<locals>.log_training_results      EPOCH_COMPLETED            19.43245
            run.<locals>.log_validation_results    EPOCH_COMPLETED             2.55271
            run.<locals>.log_time                  EPOCH_COMPLETED             0.00049
            run.<locals>.log_intermediate_results  EPOCH_COMPLETED             0.00106
            run.<locals>.log_training_loss         ITERATION_COMPLETED           0.059
            run.<locals>.log_time                  COMPLETED             not triggered
            -------------------------------------  -------------------  --------------
            Total                                                             22.04571
            -------------------------------------  -------------------  --------------
            Processing took total 11.29543s [min/index: 0.00393s/1875, max/index: 0.00784s/0,
             mean: 0.00602s, std: 0.00034s]
            Dataflow took total 16.24365s [min/index: 0.00533s/1874, max/index: 0.01129s/937,
             mean: 0.00866s, std: 0.00113s]

``` 

with such modification we can increase font-size from 62 to 72

Reply:
or 69% to fit that `...`